### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by aradradev**
Pull Request: In this project I've added the following features:

- A route.
- An action in the correct controller.
- A view file.
- Make sure that the code handles only the 4 required URLs: 

- [ ] users#index
- [ ] users#show
- [ ] posts#index
- [ ] posts#show

Please review this PR, and give your feedback if needed.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  get 'users/:id' => 'users#show', as: :user
+  get 'posts/:id' => 'posts#show', as: :post
```
Comment (Gambit142) [inline config/routes.rb]: - [ ] Generally, you ought to create four restful _**routes**_ that `GET` resources from the application:
  - One for the `Users#Index` page
  - One for the `Users#Show` page
  - One for the `Posts#Index` page and 
  - One for the `Posts#Show` page

In the schema diagram, remember that the _**Post**_ belongs to the **_User_**. This means when you are creating the routes for the Posts, there should be a way to connect the users to their posts. Kindly implement this change. 👍 😎

 
#### Hint: You can read this [article](https://guides.rubyonrails.org/routing.html#:~:text=Nested%20routes%20allow%20you%20to%20capture%20this%20relationship%20in%20your%20routing.%20In%20this%20case%2C%20you%20could%20include%20this%20route%20declaration%3A) to learn how to create **_restful resourceful nested routes_**. Remember that you ought to create only the routes that are needed. For this, you can follow the pattern used in this [article](https://guides.rubyonrails.org/routing.html#:~:text=2.7.2%20Shallow%20Nesting). 💻 🖱️

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @aradradev 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Created controllers for `User` and `Post` models ✔️ 
- Created view templates for the necessary controllers and their actions ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: config/routes.rb (hunk 1 above)
Comment (aradradev) [reply config/routes.rb]: @Gambit142, Thanks for the article man, you are a good reviewer!

**Review by ShadyShawkat — APPROVED** [review]
## Hi @aradradev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**